[2023-10-10T14:57:58.450+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow_exam.retrieve_raw_data scheduled__2023-10-10T14:56:00+00:00 [queued]>
[2023-10-10T14:57:58.475+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow_exam.retrieve_raw_data scheduled__2023-10-10T14:56:00+00:00 [queued]>
[2023-10-10T14:57:58.477+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 6
[2023-10-10T14:57:58.524+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): retrieve_raw_data> on 2023-10-10 14:56:00+00:00
[2023-10-10T14:57:58.577+0000] {standard_task_runner.py:57} INFO - Started process 248 to run task
[2023-10-10T14:57:58.615+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'retrieve_raw_data', 'scheduled__2023-10-10T14:56:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpntpq22ha']
[2023-10-10T14:57:58.648+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask retrieve_raw_data
[2023-10-10T14:57:58.776+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-10-10T14:57:58.979+0000] {task_command.py:415} INFO - Running <TaskInstance: airflow_exam.retrieve_raw_data scheduled__2023-10-10T14:56:00+00:00 [running]> on host 137abce082a9
[2023-10-10T14:57:59.180+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***_exam' AIRFLOW_CTX_TASK_ID='retrieve_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-10T14:56:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-10T14:56:00+00:00'
[2023-10-10T14:58:00.273+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-10T14:58:00.303+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=retrieve_raw_data, execution_date=20231010T145600, start_date=20231010T145758, end_date=20231010T145800
[2023-10-10T14:58:00.372+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-10T14:58:00.465+0000] {taskinstance.py:2776} INFO - 2 downstream tasks scheduled from follow-on schedule check
